src/graph/validator/MutateValidator.cpp

```diff
@@ -348,68 +350,71 @@ Status DeleteVerticesValidator::toPlan() {
 
   auto *dedupVid = Dedup::make(qctx_, nullptr);
   dedupVid->setInputVar(vidVar);
+  DeleteVertices *dvNode = nullptr;
+  if (withEdge_) {
+    std::vector<storage::cpp2::EdgeProp> edgeProps;
+    // make edgeRefs and edgeProp
+    auto index = 0u;
```

Comment:
You could iterates both edgeTypes_ and edgeNames_ instead of using the index.

Reply:
I did not modify these codes, and just move them into an if statement.